### PR TITLE
added addVehicle page

### PR DESCRIPTION
**Pull request by SousaLuis**

### CODE REVIEW

**Comment by rocambille** [comment]
make a crud and use a real form

**Comment by SousaLuis** [comment]
CRUD is  the next step on the list we just want to merge this before so we can all have the same pages and css

File: assets/styles/app.scss [hunk 1]
```diff
@@ -6,16 +6,48 @@ body {
...
     padding: 2rem;
 }
+
+.addVehicle {
+    margin-left: 5%;
+    text-align: start;
+}
+
+.addVehicle .selectBox {
+    width: 60vw;
```
Comment (rocambille) [inline assets/styles/app.scss]: no fixed width

File: assets/styles/app.scss [hunk 2]
```diff
@@ -6,16 +6,48 @@ body {
...
+    padding: 1rem;
+    margin-top: 1rem;
+}
+
+.addVehicle .inputBox{
+    width: calc(90vw - 2rem);
```
Comment (rocambille) [inline assets/styles/app.scss]: no fixed width

File: assets/styles/app.scss [hunk 3]
```diff
@@ -6,16 +6,48 @@ body {
...
+    padding: 1rem;
+    margin-top: 1rem;
+}
+
+.addVehicle .button{
+    width: 90vw;
```
Comment (rocambille) [inline assets/styles/app.scss]: no fixed width

File: assets/styles/app.scss [hunk 4]
```diff
@@ -6,16 +6,48 @@ body {
...
+    padding: 1rem;
+    position: fixed;
```
Comment (rocambille) [inline assets/styles/app.scss]: no fixed position, position: sticky may be the right choice here

File: templates/add_vehicle/addVehicle.html.twig [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            <section class="box inputBox">
+                <label for="moto-name">Nom de la moto : </label>
+                <input type="text" name="name" id="moto-name">
+            </section>
+            <section class="box inputBox">
+                <p>Nombre d'heure d'utilisation de la moto : </p>
+                <input type="text" name="usage" id="moto-usage">
+            </section>
+        </form>
+        <button class="button">Suivant</button>
```
Comment (rocambille) [inline templates/add_vehicle/addVehicle.html.twig:35]: button should in the form

File: templates/add_vehicle/addVehicle.html.twig [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+                </select>
+            </section>
+            <section class="box inputBox">
+                <label for="moto-name">Nom de la moto : </label>
+                <input type="text" name="name" id="moto-name">
+            </section>
+            <section class="box inputBox">
+                <p>Nombre d'heure d'utilisation de la moto : </p>
+                <input type="text" name="usage" id="moto-usage">
+            </section>
```
Comment (rocambille) [inline templates/add_vehicle/addVehicle.html.twig]: a section should have a title, if you have no title use div

File: assets/styles/app.scss [hunk 7]
```diff
@@ -6,16 +6,48 @@ body {
...
     margin-left: calc(20% - 2rem);
     margin-right: calc(20% - 2rem);
     margin-top: 2rem;
     margin-bottom: 2rem;
-    text-align: center;
     padding: 2rem;
 }
+
+.addVehicle {
+    margin-left: 5%;
```
Comment (rocambille) [inline assets/styles/app.scss]: use a better practice

File: assets/styles/app.scss [hunk 8]
```diff
@@ -6,16 +6,48 @@ body {
...
+    background-color: $color-3;
     border: white thin ;
+    text-align: center;
     border-radius: 4px;
+}
+
+.home .box {
+    width: 60vw;
     margin-left: calc(20% - 2rem);
     margin-right: calc(20% - 2rem);
```
Comment (rocambille) [inline assets/styles/app.scss]: use a better practice

**Review by rocambille — CHANGES_REQUESTED** [review]

**Pull request merged**